### PR TITLE
Replace duplicate development config files with symlinks.

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: tests/config/users.d/more_users.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<yandex>
+    <users>
+        <secondary>
```
Comment (azat) [inline tests/config/users.d/more_users.xml:4]: How about creating user via DDL?

**Review by azat — APPROVED** [review]
Thanks for catching this! (I saw that it fails only today, and was planning to take a look at it tonight)

File: tests/config/users.d/more_users.xml (hunk 1 above)
Comment (alexey-milovidov) [reply tests/config/users.d/more_users.xml:4]: I forgot about that possibility :laughing:

**Comment by alexey-milovidov** [comment]
Squashing is Ok.

**Pull request merged**